app/controllers/words_controller.rb

```diff
@@ -40,7 +40,7 @@ def update
 
   def destroy
     Word.find(params[:id]).destroy
-    redirect_to root_url
+    redirect_to category_words_path
```

Comment:
please fix this error:
![image](https://user-images.githubusercontent.com/73863408/141783301-7322cb10-f83b-4c28-af78-d9a794e79a38.png)

this is because there is _something missing_ for the `routes.rb` to work properly..